package.json

```diff
@@ -136,5 +141,14 @@
   },
   "resolutions": {
     "jquery": "^3.3.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix",
```

Comment:
note that eslint will run on pre-commit git hook for staged js files. This was added by default and I don't see a reason to remove. That means that can't commit files with eslint errors inside (warnings are ok)